tests/test_lava.py

```diff
@@ -0,0 +1,41 @@
+import kernelci.lava
+import pytest
+
+
+def test_get_device_type_by_name_1():
```

Comment:
Please add some blank lines between each function, ideally to pass all the `pep8` checks.

Reply:
Are the checks you expect developers to run documented or automated? What check specifically should I run to ensure compliance? Incidentally, I vote for 'black' when it comes to python formatting. It does create a lot of whitespace in tests though, as it puts two empty lines between each function.

Reply:
I was referring to the usual coding standard for Python [1] as I thought that's what had been adopted by default by the Python community.  If you want to run "black" instead then why not, I'm not familiar with it.

As far as this PR is concerned, `pep8` only reports these things with the newly added files:
```
$ pep8 tests/test_lava.py kernelci/lava.py 
tests/test_lava.py:6:68: E711 comparison to None should be 'if cond is None:'
tests/test_lava.py:10:72: E711 comparison to None should be 'if cond is None:'
tests/test_lava.py:30:80: E501 line too long (82 > 79 characters)
tests/test_lava.py:31:9: E711 comparison to None should be 'if cond is None:'
```

The comparison with `None` seems fair since there's only one `None` object in Python.  Regarding line lengths, there's no absolute solution but I personally prefer to follow the 79 characters rule.

So for this change maybe fix the `None` items, and the 79 characters if you agree with them, and then create a PR or Github issue to discuss the topic to get an agreement going forward?

[1] https://www.python.org/dev/peps/pep-0008/

Reply:
Thanks - fixed all those things and updated the PR. I'll follow up in a separate PR with either pycodestyle (pep8) or black integration.

Reply:
Thanks!